src/b-splines/prefiltering.jl

```diff
@@ -78,3 +78,35 @@ function prefilter!{TWeights,TCoefs<:AbstractArray,IT<:Tuple{Vararg{Union{BSplin
 end
 
 prefiltering_system(::Any, ::Any, ::Any, ::Any, ::Any) = nothing, nothing
```

Comment:
@timholy is this method actually good for anything? It seems to me that it would be better to get MethodErrors on prefiltering_system than on whatever happens to use it...